ts/src/coder/accounts.ts

```diff
@@ -50,9 +50,13 @@ export class AccountsCoder<A extends string = string> {
     }
     return layout.decode(data);
   }
-}
 
-// Calculates unique 8 byte discriminator prepended to all anchor accounts.
-export async function accountDiscriminator(name: string): Promise<Buffer> {
-  return Buffer.from(sha256.digest(`account:${name}`)).slice(0, 8);
+  /**
+   * Calculates and returns a unique 8 byte discriminator prepended to all anchor accounts.
+   *
+   * @param name The name of the account to calculate the discriminator.
+   */
+  public static accountDiscriminator(name: string): Buffer {
```

Comment:
The original method returned `Promise<Buffer>` but Buffer.from doesn't return a Promise so I removed it, if it's needed for some reason I can put it back.

Reply:
I figured it was something around those lines.